docs/sources/features/reporting.md

```diff
@@ -27,7 +27,12 @@ With Reports there are a few things to keep in mind, most importantly, any chang
 
 > SMTP must be configured for reports to be sent
 
-We recommend using the new image rendering plugin with reporting as it supports a wider range of panels than the built-in image rendering. Read more about it [here]({{< relref "../administration/image_rendering.md#grafana-image-renderer-plugin" >}})
+
+### Rendering
+
+> Reporting requires the [rendering plugin]({{< relref "../administration/image_rendering.md#grafana-image-renderer-plugin" >}}).
```

Comment:
These two paragraphs are redundant. Combine and refine.

Reply:
Is this a comment on #20783?  I think this was only included from a merge?  and then I think it somehow automatically requested a review from you ¯\_(ツ)_/¯

If the comment is relevant -- can you find somewhere appropriate or it?  Same with the other comment -- this PR is not looking at rendering or debian

Reply:
I am automatically added as a reviewer whenever something from the docs repo is changed.

Reply:
cool -- I guess you were added to this because someone touched a file sometime and that got merged into this... even though I never touched the file.  Seems weird, but sure.  (to be clear this PR never touched any docs files!)